content/en/docs/concepts/security/index.md

```diff
@@ -632,331 +629,267 @@ authorization result, `ALLOW` or `DENY`.
-to all services in the `default` namespace.
+The following example shows an `AuthorizationPolicy` that allows two sources
+(service account `cluster.local/ns/default/sa/sleep` and namespace `dev`) to access the
+workloads with labels `app: httpbin` and `version: v1` in namespace foo when the request
+is sent with a valid JWT token.
 
 {{< text yaml >}}
-apiVersion: "rbac.istio.io/v1alpha1"
-kind: ServiceRole
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
 metadata:
-  name: service-admin
-  namespace: default
+ name: httpbin
+ namespace: foo
 spec:
-  rules:
-  - services: ["*"]
+ selector:
+   matchLabels:
+     app: httpbin
+     version: v1
+ rules:
+ - from:
+   - source:
+       principals: ["cluster.local/ns/default/sa/sleep"]
+   - source:
+       namespaces: ["dev"]
+   to:
+   - operation:
+       methods: ["GET"]
+   when:
+   - key: request.auth.claims[iss]
+     values: ["https://accounts.google.com"]
 {{< /text >}}
 
-Here is another role: `products-viewer`, which has read, `"GET"` and `"HEAD"`,
-access to the service `products.default.svc.cluster.local` in the `default`
-namespace.
+#### Scope and workload selector
+
+Policy scope (target) is determined by `metadata/namespace` and an optional `selector`.
+
+The `metadata/namespace` tells which namespace the policy applies. If set to the
+root namespace, the policy applies to all namespaces in a mesh. The value of
+root namespace is configurable, and the default is `istio-config`. If set to a
+normal namespace, the policy will only apply to the specified namespace.
+
+A workload `selector` can be used to further restrict where a policy applies.
+The `selector` uses pod labels to select the target workload. The workload
+selector contains a list of `{key: value}` pairs, where the `key` is the name of the label.
+If not set, the authorization policy will be applied to all workloads in the same namespace
+as the authorization policy.
+
+The following example policy `allow-read` which allows `"GET"` and `"HEAD"` access to
+the workload with label `app: products` in the `default` namespace.
 
 {{< text yaml >}}
-apiVersion: "rbac.istio.io/v1alpha1"
-kind: ServiceRole
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
 metadata:
-  name: products-viewer
+  name: allow-read
   namespace: default
 spec:
+  selector:
+    matchLabels:
+      app: products
   rules:
-  - services: ["products.default.svc.cluster.local"]
-    methods: ["GET", "HEAD"]
+  - to:
+    - operation:
+         methods: ["GET", "HEAD"]
 {{< /text >}}
 
-In addition, we support prefix matching and suffix matching for all the fields
-in a rule. For example, you can define a `tester` role with the following
-permissions in the `default` namespace:
+#### Value matching
+
+Exact match, prefix match, suffix match, and presence match are supported for most
+of the field with a few exceptions (e.g., "when/key" only supports exact match).
```

Comment:
I think `ip` and `port` are also not supported

Reply:
Let's clearly call out these fields only support exact match, either in this doc, or document it in API reference. So that people do not need to guess.